src/index.ts

```diff
@@ -100,6 +100,15 @@ export default function docusaurusOpenRpc(
         }),
       );
       if (loadedVersions === undefined) {
+        actions.addRoute({
+          path: join(context.baseUrl, options.path, 'json-rpc-api'),
```

Comment:
Maybe we should take the `'json-rpc-api'` as a plugin option to make the url configurable?

Reply:
added a ticket for that here:

https://github.com/MetaMask/docusaurus-openrpc/issues/58